### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by vinjana**
* sort-log filename fix
* qc-json filename fix in error message
* added licence information

### CODE REVIEW

File: resources/analysisTools/qcPipeline/fakeDupmarkMetrics.pl [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/perl
+#
+# Copyright (c) 2018 German Cancer Research Center (DKFZ).
```
Comment (dankwart-de) [inline resources/analysisTools/qcPipeline/fakeDupmarkMetrics.pl]: If you change it, you can update the date to 2019.

File: resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl [hunk 2]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/perl
+#
+# Copyright (c) 2018 German Cancer Research Center (DKFZ).
```
Comment (dankwart-de) [inline resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl]: Again. But it is not a must.

File: resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl [hunk 3]
```diff
@@ -298,11 +303,22 @@
 		$devsqsum += $howmany * (($i - $average)**2);
 	}
 
-	my $stddev = sqrt($devsqsum/($ctr-1));
-	my $sdrd=int($stddev+0.5);
+	my $stddev;
+	my $sdrd;
+	my $sdperc;
+	my $rounded;
+	if ($ctr - 1 == 0) { # Cannot calculate SD and derived values due to lack of data.
```
Comment (dankwart-de) [inline resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl]: If you want it a bit shorter, you could assign the "NA" above and invert the if condition. Saves some lines and might be a bit more readable.

File: resources/analysisTools/qcPipeline/qcJson.pl [hunk 4]
```diff
@@ -329,20 +329,20 @@ ($$)
 
 
 sub ensureInt ($) {
-    my ($val) = @_;
-    if ($val !~ /^\d$/) {
+	my ($val) = @_;
+	if ($val !~ /^(?:\d|NaN|NA)$/) {
```
Comment (dankwart-de) [inline resources/analysisTools/qcPipeline/qcJson.pl]: Could you comment, what this regex does? Also the regex below? 
Do I understand it right, that this one checks the complete string to be either a decimal or NaN or NA?

File: resources/analysisTools/qcPipeline/qcJson.pl [hunk 5]
```diff
@@ -329,20 +329,20 @@ ($$)
...
+	}
+	return $val;
 }
 
 
 sub ensureFloat ($) {
-    my ($val) = @_;
-    if ($val !~ /^(?:-?\d+(?:\.\d+)?|\d(?:\.\d+)?(?:[eE]-?\d+)?)$/) {
+	my ($val) = @_;
+	if ($val !~ /^(?:-?\d+(?:\.\d+)?|\d(?:\.\d+)?(?:[eE]-?\d+)?|NaN|NA)$/) {
```
Comment (dankwart-de) [inline resources/analysisTools/qcPipeline/qcJson.pl]: This checks for either a decimal followed by a point with decimals OR a decimal followed by exponent OR NaN OR NA?

File: resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl (hunk 3 above)
Comment (vinjana) [reply resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl]: done

File: resources/analysisTools/qcPipeline/qcJson.pl (hunk 5 above)
Comment (vinjana) [reply resources/analysisTools/qcPipeline/qcJson.pl]: yip

**Pull request merged**